lightsaber/processor/src/main/java/com/joom/lightsaber/processor/validation/DependencyResolver.kt

```diff
@@ -181,6 +182,15 @@ class DependencyResolverImpl(
     for (factoryProvisionPoint in factory.provisionPoints) {
       add(factoryProvisionPoint, path.with(factoryProvisionPoint))
     }
+    if (includeAllDependenciesInGraph) {
+      for (factoryProvisionPoint in factory.provisionPoints) {
+        for (injectee in factoryProvisionPoint.injectionPoint.injectees) {
+          if (injectee is FactoryInjectee.FromInjector) {
+            dependencyGraph.put(factory.dependency, injectee.dependency)
+          }
+        }
+      }
+    }
```

Comment:
Maybe this code should be moved to the `add` method called above?

Reply:
Fixed